extra.gradle

```diff
@@ -0,0 +1,149 @@
+buildscript {
```

Comment:
I think you can safely move all the content of this script to `build.gradle`. Studio does not rewrite it anymore, see http://files.cuba-platform.com/cuba/release-notes/6.8/#build_scripts

Reply:
moved to build.gradle